README.md

```diff
@@ -1050,7 +1050,9 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Musa Opaluwa](https://github.com/musaa99)
   - [Musab Habeeb](https://github.com/Musab1258)
   - [Musibau Anifowose Bolaji](https://github.com/adigson)
+  - [Muskan Singh](https://github.com/Radiosnack)
   - [Muthu Annamalai V.](https://github.com/muthuannamalai12)
+  
```

Comment:
remove this!

Reply:
Line 1055 ryt ? I have removed line 1055